source/operations/cert-manager.rst

```diff
@@ -0,0 +1,577 @@
+
+- ``minio-operator``
+- ``tenant-1``
+- ``tenant-2``
+- ``other-namespace``
+
+.. image:: /images/k8s/cert-manager-namespaces.svg
+   :width: 600px
+   :alt: A graphic depiction of a kubernetes cluster with four namespaces. Each namespace in its own white box. One is labeled minio-operator with contents two sts content items. A second is labeled tenant-1 with a MinIO logo and four drives of a pool. A third is labeled tenant-2 with similar contents to the tenant-1 box. The fourth is labeled other namespace with contents of other pods.
+   :align: center
+
+This guide shows you how to set up a different Certificate Authority (CA) in each namespace.
+Each namespace's CA references the global ``Cluster Issuer``.
+
+
+The following graphic depicts how various namespaces make use of either an ``Issuer`` or ``ClusterIssuer`` type.
+
+- cert-manager is installed in the ``cert-manager`` namespace, which does not have either an ``issuer`` or a ``ClusterIssuer``.
+- The ``default`` namespace receives the global ``Cluster Issuer``.
+- Each tenant's namespace receives a local ``Issuer``.
+- The ``minio-operator`` namespace receives a local ``Issuer``. 
+  More about services that require TLS certificates in the ``minio-operator`` namespace are covered :ref:`below <minio-operator-services-with-cert-manager>`.
+
+.. image:: /images/k8s/cert-manager-cluster.svg
+   :width: 600px
+   :alt: A graphic depiction of a kubernetes cluster with five separate namespaces represented by different boxes. One is labeled minio-operator with a text box that says "minio-operator: issuer". A second is labeled tenant-1 with text box that says "tenant-1: issuer". A third is labeled default with a text box that says "root:ClusterIssuer". A fourth is labeled tenant-2 with a text box that says "tenant-2: issuer". The fifth is labeled cert-manager with a text box that says "minio-operator: issuer".
+   :align: center
+
+.. note::
+
+   This guide uses a self-signed ``Cluster Issuer``. 
+   You can also use `other Issuers supported by cert-manager <https://cert-manager.io/docs/configuration/issuers/>`__.
+   The main difference is that you must provide the ``Issuer`` CA certificate to MinIO, instead of the CA's mentioned in this guide.
+
+
+Prerequisites
+-------------
+
+- A `supported version of Kubernetes <https://kubernetes.io/releases/>`__. 
+ 
+  While cert-manager supports `earlier K8s versions <https://cert-manager.io/docs/installation/supported-releases/>`__, the MinIO Operator requires a currently supported version.
+- `kustomize <https://kustomize.io/>`__ installed
+- ``kubectl`` access to your ``k8s`` cluster
+
+Getting Started
+---------------
+
+Setup cert-manager
+~~~~~~~~~~~~~~~~~~
+
+Install cert-manager. 
+`Release 1.12.X LTS <https://cert-manager.io/docs/releases/release-notes/release-notes-1.12/>`__ is preferred, but you may install latest.
+
+The following command installs version 1.12.13 using ``kubectl``.
```

Comment:
Worth doing a substitution here in case the version bumps later?

Reply:
Perhaps? Not sure how critical that is to stay up. Mostly we want this lineage. Which particular release within 1.12.x is less important from our perspective.